hazelcast-enterprise/Dockerfile

```diff
@@ -38,26 +42,29 @@ RUN echo "Updating Alpine system" \
     && mkdir lib \
     && curl -sf -o ${HZ_HOME}/lib/${HZ_JAR} \
          -L https://repository.hazelcast.com/release/com/hazelcast/hazelcast-enterprise-all/${HZ_VERSION}/${HZ_JAR} \
-   && echo "Installing JCache" \
-   && curl -sf -o "${HZ_HOME}/lib/${CACHE_API_JAR}" \
+    && echo "Installing JCache" \
+    && curl -sf -o "${HZ_HOME}/lib/${CACHE_API_JAR}" \
          -L "https://repo1.maven.org/maven2/javax/cache/cache-api/${CACHE_API_VERSION}/${CACHE_API_JAR}" \
-   && echo "Installing Hazelcast plugins" \
-   && mvn -f dependency-copy.xml \
+    && echo "Installing Hazelcast plugins" \
+    && mvn -f dependency-copy.xml \
            -Dhazelcast-kubernetes-version=${HZ_KUBE_VERSION} \
            -Dhazelcast-eureka-version=${HZ_EUREKA_VERSION} \
            -Dnetty.version=${NETTY_VERSION} \
            -Dnetty-tcnative.version=${NETTY_TCNATIVE_VERSION} \
            dependency:copy-dependencies \
-   && echo "Maven clean-up" \
-   && apk del maven \
-   && rm -rf ~/.m2 \
-   && rm -f dependency-copy.xml \
-   && echo "Granting read permission to ${HZ_HOME}" \
-   && chmod -R +r $HZ_HOME \
-   && echo "Setting Pardot ID to 'docker'" \
-   && echo 'hazelcastDownloadId=docker' > hazelcast-download.properties \
-   && echo "Cleaning APK packages" \
-   && rm -rf /var/cache/apk/*
+    && echo "Maven clean-up" \
+    && apk del maven \
+    && rm -rf ~/.m2 \
+    && rm -f dependency-copy.xml \
+    && echo "Installing JMX Prometheus Agent" \
+    && curl -sf -o "${HZ_HOME}/lib/jmx_prometheus_javaagent.jar" \
+         -L "https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/${JMX_PROMETHEUS_AGENT_VERSION}/jmx_prometheus_javaagent-${JMX_PROMETHEUS_AGENT_VERSION}.jar" \
+    && echo "Granting read permission to ${HZ_HOME}" \
+    && chmod -R +r $HZ_HOME \
+    && echo "Setting Pardot ID to 'docker'" \
+    && echo 'hazelcastDownloadId=docker' > hazelcast-download.properties \
+    && echo "Cleaning APK packages" \
+    && rm -rf /var/cache/apk/*
```

Comment:
is there any change here or it is just formatting?

Reply:
Just formatting.